client/src/constants/index.js

```diff
@@ -154,6 +154,25 @@ export const Sections = {
       name: "Quiz",
     },
   },
+  8: {
+    fullname: "Algorithmic Bias",
```

Comment:
We should make a todo comment here to update the full name when we have officially decided on a title for the lab